lib/CModHandler.h

```diff
@@ -268,20 +268,22 @@ class DLL_LINKAGE CModHandler
 		int MAX_HEROES_AVAILABLE_PER_PLAYER;
 		int MAX_HEROES_ON_MAP_PER_PLAYER;
 		bool WINNING_HERO_WITH_NO_TROOPS_RETREATS;
+		bool BLACK_MARKET_MONTHLY_ARTIFACTS_CHANGE;
 
 		template <typename Handler> void serialize(Handler &h, const int version)
 		{
 			h & data & CREEP_SIZE & WEEKLY_GROWTH & NEUTRAL_STACK_EXP & MAX_BUILDING_PER_TURN;
 			h & DWELLINGS_ACCUMULATE_CREATURES & ALL_CREATURES_GET_DOUBLE_MONTHS &
 			MAX_HEROES_AVAILABLE_PER_PLAYER & MAX_HEROES_ON_MAP_PER_PLAYER;
 			if(version >= 756)
-			{

```

Comment:
If you didn't do that yourself please disable this automatic curly brackets removal. In many cases it's makes sense to avoid or remove them, but serialization compatibility code is not one of those places.